Resources/Prototypes/Atmospherics/reactions.yml

```diff
@@ -14,14 +14,12 @@
   id: TritiumFire
   priority: -1
   minimumTemperature: 373.149 # Same as Atmospherics.FireMinimumTemperatureToExist
-  minimumRequirements: # In this case, same as minimum mole count.
-    - 0.01  # oxygen
-    - 0     # nitrogen
-    - 0     # carbon dioxide
-    - 0     # plasma
-    - 0.01  # tritium
-  effects:
-    - !type:TritiumFireReaction {}
+  enthalpy: 284000
+  reactants:
+    4: 2 # 2 Tritium
+    0: 1 # 1 Oxygen
+  products:
+    5: 2 # 2 Water
```

Comment:
You should be able to use the Gas enum name for the key here.

Reply:
This is very cool, I'm converting the reactions to use the gas enum name and removing the comments. This really helps with read and maintainability. Perhaps this is a good reason to keep it as a dict and convert it on load in a future accelerated version.